testdata/slsa_v02_provenance.json

```diff
@@ -12,20 +12,20 @@
     "predicate": {
         "builder": {
             "id": "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@refs/tags/v1.2.0"
-        },
+                },
```

Comment:
(nit) the indentation seems incorrect now?

Reply:
Thanks for noticing this. Fixed.